### PR TITLE
Feature/ij 28 homepage

### PR DESCRIPTION
**Pull request by AlexW555**
A Basic homepage, I will add another ticket on Jira to take notes on what improvements to be made:
- Explaining about the app
- Main features
- Useful links

### CODE REVIEW

File: app/views/pages/main.html.erb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  <header class="mb-auto">
+    <div>
+      <h3 class="float-md-start mb-0">Include UK</h3>
+      <nav class="nav nav-masthead justify-content-center float-md-end">
+        <a class="nav-link active" aria-current="page" href="#">Home</a>
+        <a class="nav-link" href="#">Features</a>
+        <a class="nav-link" href="#">Contact</a>
+      </nav>
+    </div>
+  </header>
```
Comment (phil-6) [inline app/views/pages/main.html.erb:17]: This duplicates `app/views/layouts/application.html.erb`

File: app/views/pages/main.html.erb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      <a href="#" class="btn btn-lg btn-secondary fw-bold border-white bg-white text-black-50">Open the App</a>
+    </p>
+  </main>
+
+  <footer class="mt-auto text-white-50">
+    <p>Produced by the <a href="https://legaltech.wales/" class="text-white">Legal Innovation Lab Wales</a></p>
+  </footer>
+</div>
+</body>
+</html>
```
Comment (phil-6) [inline app/views/pages/main.html.erb:33]: as per above comment, this will end up being duplicate code I think.

**Review by phil-6 — CHANGES_REQUESTED** [review]

**Comment by IeuanSkinner** [comment]
So when the user or team member hits the homepage they'll need to be able to navigate to the relevant login screen, could we have 2 buttons side-by-side for 'Include UK Staff' and 'User' (with icons?) that redirects them to /team_members/sign_in and /users/sign_in?

**Comment by AlexW555** [comment]
> So when the user or team member hits the homepage they'll need to be able to navigate to the relevant login screen, could we have 2 buttons side-by-side for 'Include UK Staff' and 'User' (with icons?) that redirects them to /team_members/sign_in and /users/sign_in?

Yep, great idea!

File: app/assets/stylesheets/homepage.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
+// Place all the styles related to the pages controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
+
+@import "variables";
+
+body#overboot {
+  font-family: $font;
```
Comment (IeuanSkinner) [inline app/assets/stylesheets/homepage.scss:8]: This is redundant, it's already in global

**Review by phil-6 — APPROVED** [review]

**Pull request merged**